src/extra/normalize.css

```
@@ -283,6 +287,10 @@
     background-color: var(--surface-2);
   }
 
+  @nest [data-theme=dark] & {
```

nice! looks like you found your way. 

for consistency, can you put the attribute values in quotes?

```suggestion
  @nest [data-theme="dark"] & {
```